admin/users/views.py

```diff
@@ -61,15 +61,11 @@ def delete(self, request, *args, **kwargs):
                     action_flag=flag
                 )
         except AttributeError:
```

Comment:
This is doing duck-typing. A `None` return value doesn't quack. This was a major part of refactors after users were refactored. I could change this in all previous/subsequent modules...